### DISCUSSION TITLE
Migrate VM encountering Cannot recv data: Host key verification failed: Connection reset by peer

### DISCUSSION THREAD

**Discussion by Robpeter** [cockpit-project/cockpit-machines#1499] [General]
I saw this [https://github.com/cockpit-project/cockpit-machines/issues/684](https://github.com/cockpit-project/cockpit-machines/issues/684) regarding an error I'm getting when migrating VM to another host.
`Cannot recv data: Host key verification failed: Connection reset by peer`

Is there documentation on how to migrate a vm to another host with Cockpit-Machines?
Is it the logged in user's ssh key exchange? Or is it an id that Cockpit is running as?

**Comment by mac2net** [comment]
I suggest use the modern version of [sneaker.net](http://sneaker.net/) and copy the file and re-bless the image at the other end. It works and also it works. You just have to copy the following to the same paths at the destination:

/etc/libvirt/qemu/serv01.xml
/var/lib/libvirt/images/serv01.img
Then at the destination end bless it:

virsh define /etc/libvirt/qemu/serv01.xml 
There is also a process to “clean” the image of your deep secrets

virt-sysprep: reset or unconfigure a virtual machine so clones can be made
It has a lot of options. It looks complicated but cleaning the basics is easy.
